sync.cmd

```diff
@@ -25,22 +27,26 @@ if /I [%1]==[/?] goto Usage
 
 if /I [%1] == [/p] (
     set packages=true
-    set thisArgs=!thisArgs!%1
+    set thisArgs=%thisArgs%%1
     goto Next
 )
 
 if /I [%1] == [/s] (
     set src=true
-    set thisArgs=!thisArgs!%1
+    set thisArgs=%thisArgs%%1
     goto Next
 )
 
-if [!thisArgs!]==[] (
-  set unprocessedBuildArgs=!allargs!
-) else (
-  call set unprocessedBuildArgs=%%allargs:*!thisArgs!=%%
+if /I [%1] == [/t] (
```

Comment:
I'm changing the structure and things inside sync.cmd, so if we need to pass extra parameters (like `PackageDrops`) to msbuild, please add them.